cmd/otel-agent/command/command.go

```diff
@@ -63,12 +64,18 @@ func makeCommands(globalParams *subcommands.GlobalParams) *cobra.Command {
 }
 
 const configFlag = "config"
+const coreConfigFlag = "core-config"
+const syncDelayFlag = "sync-delay"
+const syncTOFlag = "sync-to"
 
 func flags(reg *featuregate.Registry, cfgs *subcommands.GlobalParams) *flag.FlagSet {
 	flagSet := new(flag.FlagSet)
 
 	flagSet.Var(cfgs, configFlag, "Locations to the config file(s), note that only a"+
 		" single location can be set per flag entry e.g. `--config=file:/path/to/first --config=file:path/to/second`.")
+	flagSet.StringVar(&cfgs.CoreConfPath, coreConfigFlag, "", "Location to the Datadog Agent config file.")
+	flagSet.DurationVar(&cfgs.SyncDelay, syncDelayFlag, 0, "Delay before first config sync.")
+	flagSet.DurationVar(&cfgs.SyncTO, syncTOFlag, 3*time.Second, "Timeout for sync requests.")
```

Comment:
Can we rename `SyncTo` to `SyncTimeout`. I believe it is more descriptive that way